metricbeat/docs/modules/mssql/db.asciidoc

```diff
@@ -0,0 +1,23 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[metricbeat-metricset-mssql-db]]
+=== mssql db metricset
+
+experimental[]
+
+include::../../../module/mssql/db/_meta/docs.asciidoc[]
+
+
+==== Fields
+
+For a description of each field in the metricset, see the
+<<exported-fields-mssql,exported fields>> section.
```

Comment:
The key `exported-fields-mssql` does not exist here. Probably the `generate_fields_docs` scripts needs an update here on line 160 to not read the `fields.yml` from the oss beat but from the x-pack version. Perhaps you can open a separate PR for this and rebase this one on top of it. Best also check how Filebeat does it as I assume there it's already working. Perhaps only the path passed to the method must be adjusted.